README.md

```diff
@@ -1,17 +1,20 @@
 # Lisk SDK examples
 
-Example applications using Alpha SDK.
+This project contains example blockchain applications using the latest Lisk SDK.
+Each example project represents an industry use case in which we showcase the potential of the Lisk SDK and how it can transform this industry.
 
-## [Transport Workshop](./transport/Workshop2.adoc)
-Follow the Lisk Transport workshop which has been designed for the **Lisk.js 2019** event!
+Currently, we have two projects ready:
 
-## [Hello World](./hello_world/README.md)
-A hello world application of Alpha SDK. Guides through the basic flow of using Alpha SDK. By creating a new custom transaction with no practical use case, you can get an idea of why and how to implement the required methods of the custom transaction interface.
+- Hello world (beginner example)
+- Lisk Transport (**Supply chain industry use case**)
 
-## [Cashback](./cashback/README.md)
+All industry use cases will be kept up to date with the latest Lisk SDK release.
+Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases. 
+At the moment, the `/archive` folder contains examples for Lisk SDK `version 2.3.5`.
 
-The example of a simple but valid use-case - cashback. On top of the regular balance transfer, the example implements the cashback functionality. 
+## [Supply chain: Lisk Transport](./transport/Workshop2.adoc)
+Originally, the Lisk Transport workshop has been designed for the **Lisk.js 2019** event. The project represents a flow where a carrier can sign up to pick up a packet and deliver it to the right person in a decentralized way. A custom transaction helps with locking the fee for the drive and a warranty in case the package gets damaged or lost. In addition, anyone can become a carrier. For example, you can pick up a packet that has its destination on your daily commute and earn a small reward for this.
 
-### Interacting with a network
-In the [scripts](./scripts) directory you can find implementation of the creating, signing and sending custom transactions to the network. 
 
+## [Hello World](./hello_world/README.md)
+A hello world application of the Alpha SDK. Guides through the basic flow of using Alpha SDK. By creating a new custom transaction with no practical use case, you can get an idea of why and how to implement the required methods of the custom transaction interface.
```

Comment:
I think we stopped using the expression `Alpha SDK` and just refer to the Lisk SDK instead.

```suggestion
A hello world application of the Lisk SDK. Guides through the basic flow of using the SDK. By creating a new custom transaction with no practical use case, you can get an idea of why and how to implement the required methods of the custom transaction interface.
```